ReactiveCocoa/Swift/Signal.swift

```diff
@@ -716,6 +716,67 @@ public func tryMap<T, U, E>(operation: T -> Result<U, E>)(signal: Signal<T, E>)
 	}
 }
 
+/// Throttle values sent by the receiver, so that at least `interval`
+/// seconds pass between each, then forwards them on the given scheduler.
+///
+/// If multiple values are received before the interval has elapsed, the
+/// latest value is the one that will be passed on.
+///
+/// If the input signal completes while a value is being throttled, that value
+/// will be discarded and the returned signal will complete immediately.
+public func throttle<T, E>(interval: NSTimeInterval, onScheduler scheduler: DateSchedulerType)(signal: Signal<T, E>) -> Signal<T, E> {
+	precondition(interval >= 0)
+
+	return Signal { observer in
+		let state: Atomic<ThrottleState<T>> = Atomic(ThrottleState())
```

Comment:
This is the style that the other GitHub folks want to follow, keeping the type clarification separate from the initialization.